src/main/java/edu/kis/vh/nursery/stackImplementation/IntArrayStack.java

```diff
@@ -1,9 +1,7 @@
-package edu.kis.vh.nursery;
-
-import edu.kis.vh.nursery.list.Stack;
+package edu.kis.vh.nursery.stackImplementation;
 
 public class IntArrayStack implements Stack {
-    private static final int ERROR = -1;
+    private static final int ERROR = 0;
```

Comment:
ERROR: total variable initialization and validation should always remain -1.
The default return value should be moved to the interface (3.1.8).